packages/ui/skeleton/index.tsx

```diff
@@ -1,15 +1,19 @@
 import classNames from "@calcom/lib/classNames";
 
 type SkeletonBaseProps = {
+  /** @deprecated @see https://tailwindcss.com/docs/content-configuration#dynamic-class-names */
   width?: string;
+  /** @deprecated @see https://tailwindcss.com/docs/content-configuration#dynamic-class-names */
   height?: string;
   className?: string;
 };
 
 interface AvatarProps extends SkeletonBaseProps {
-  // Limit this cause we don't use avatars bigger than thi
-  width: "2" | "3" | "4" | "5" | "6" | "8" | "12";
-  height: "2" | "3" | "4" | "5" | "6" | "8" | "12";
+  // Limit this cause we don't use avatars bigger than this
+  /** @deprecated @see https://tailwindcss.com/docs/content-configuration#dynamic-class-names */
```

Comment:
why show deprecated when we just replace it everywhere?

Reply:
Not worth the effort now as we will remove it post v2